### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_Createnewservergroup_ScreenReader: ORCA screen reader is not announcing "Required" for "Server group name" and "Group description" edit fields.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio-mysql#176]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11ySev3; #A11yTCS; #DesktopApp; #GH_MySQLextensionforAzureDataStudio_Linux_Mar23; #MySQL extension for Azure Data Studio; #WCAG3.3.2; #Linux; #ScreenReader; #ORCA;

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to "New Server Group" icon and hit ENTER key.
3. "Add server group" screen gets open. Start screen reader and TAB to "Server group name" and "Group description".
4. Observe that ORCA screen reader is not announcing "Required" for "Server group name" and "Group description" edit fields.

### Actual Result:​
ORCA screen reader is not announcing "Required" for "Server group name" and "Group description" edit fields.

### Expected Result:​
ORCA screen reader should announce "Required" for "Server group name" and "Group description" edit fields as without filling those field we are not able to save the data.

### User Impact:
Screen reader users will not get information about the required field as its not announcing it as required.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachment:

[Video 3-30 at 12.37 (2).webm](https://user-images.githubusercontent.com/90901519/228807508-97a893ac-abd2-4989-a800-c19a39a663d9.webm)

**Comment by nitishsharma25** [comment]
This issue is external to MySQL extension as it is a common UI of Azure Data Studio.

**Comment by PoojaNamde** [comment]
Logged External bug: https://github.com/microsoft/azuredatastudio/issues/22691

As per the [External Bug Process - First Party Dependencies](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDV8MDF8fGM4NTQxMGIxZGFhYjQwMTZmNTIzMDhkYWM5NDgyMWQwfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgwNDM2MDg4Njg1MjIxNDZ8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxNVFkyT0RjMk5EQTROVFkzT1RzeE5qWTROelkwTURnMU5qYzVPekU1T2pjelpqUTNOV00zTFRnNU5Ea3RORFprT1MxaFlURTVMVGN3WlRobU9EWm1ZVGszT0Y4M1lqbG1ZVFV5TmkweVptRXhMVFF5WmpRdFltVXhOeTFsTXpRME5EbGpabVkyTlRSQWRXNXhMbWRpYkM1emNHRmpaWE09fDA5YTBhOTU1NzM0MDQ5NTM1OTJiMDhkYWM5NDgyMWQwfGZlNmZiOTdlMzBhNTRjNzJhNTRhNTU0Y2RjNDg0NzM3&sdata=ZVRCdGg0WldPVlRVQ2RveFFNV1FJUEc0bHc1a2QvWTExbkVLS3V5emQzYz0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-ramakatare%40microsoft.com) assigning the bug to the Product Owner. Please assign this bug back to the tester once the "External" issue has been resolved and changes are pushed to the environment that will be available for bug verification.

**Comment by keerthiduvvuri** [comment]
GithubTags:#ResolvedByTT
